js/script.js

```diff
@@ -54,6 +55,8 @@ function createTask (event){
   let taskImportance = document.querySelector(
     'input[name="taskImportance"]:checked'
   );
+  const taskId = Math.random().toString(36).substring(2, 12);
```

Comment:
How else would we create tasks?
We should be able to use <template> within this function, if we want to if that's what you mean?
The issue with the original id generation, was that (I think) any items with the same name would have the same id, so I didn't think it would solve the problem that we had.